WordPress/src/main/res/layout/reader_recommended_blog_item.xml

```diff
@@ -89,4 +76,29 @@
         app:layout_constraintTop_toBottomOf="@+id/site_name"
         tools:text="legalnomads.com" />
 
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/site_description"
+        style="@style/ReaderTextView.Site.Description"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_large"
+        android:includeFontPadding="false"
+        app:layout_constraintBottom_toBottomOf="@id/guideline_bottom"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@+id/site_icon"
+        tools:text="@tools:sample/lorem/random" />
+
+    <org.wordpress.android.ui.reader.views.ReaderFollowButton
+        android:id="@+id/site_follow_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/reader_btn_follow"
+        android:padding="@dimen/margin_extra_large"
+        app:iconTint="@color/secondary_gray_20_selector"
```

Comment:
@ashiagr I was able to use `ReaderFollowButton` 
the only thing I had to modify was an `iconTint`. For now, I just used it here directly but we could:
- create a different style
add
- add a custom attribute to set the color 
Do you have a preference?

Reply:
We have a style for "icon + caption" follow new button [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/reader_styles.xml#L241). Thinking in terms of consistency, we can use a different style for "icon only" button.

Reply:
Done :)